# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libsdl_perl/console_fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./SDL_perl.xs	2003-01-03 06:14:35.000000000 +0100
+++ ./SDL_perl.xs	2003-05-26 16:04:15.000000000 +0200
@@ -67,7 +67,7 @@
 
 #ifdef HAVE_SDL_CONSOLE
 #include "CON_console.h"
-#include "CON_consolecommands.h"
+/*#include "CON_consolecommands.h"*/
 #include "DT_drawtext.h"
 #endif
 
